README.md

```diff
@@ -1,2 +1,7 @@
 # bootstrap
+
 Foundation Bootstrap Team
+
+## Making proposals
+
+Anyone may submit an idea for a policy or program following the [staging process](STAGING_PROCESS.md)
```

Comment:
Can you close this sentence with a .?

Should we link this to CONTRIBUTING.md as well?